### PR TITLE
Changeing lib and drawing better scaled hands

### PR DESCRIPTION
**Pull request by TStod**
For review. Changes may need to be made to your folder.
See:
https://developer.leapmotion.com/documentation/java/devguide/Leap_Processing.html

### CODE REVIEW

**Comment by TStod** [comment]
Assign back and label verify when ready for merge

**Comment by TStod** [comment]
1440x2560

image

800x450

asdf

**Comment by TStod** [comment]
Overall interaction is much smoother also

File: Capstone.pde [hunk 1]
```diff
@@ -16,41 +13,20 @@ void settings() {
 }
```
Comment (jebuschrast) [inline Capstone.pde]: blah

**Comment by jebuschrast** [comment]
10/10, 11/10 with rice

**Pull request merged**